opentelemetry/instrumentation/otel-collector-config.yaml

```diff
@@ -17,6 +17,18 @@ receivers:
           layout: '%Y-%m-%dT%H:%M:%S.%fZ'
         severity:
           parse_from: body.severity
+          preset: none
+          # parse minimal set of severity strings that Cloud Logging explicitly supports
+          # https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity
+          mapping:
+            debug: debug
+            info: info
+            info3: notice
```

Comment:
The OTel levels used here are matching how googlecloud exporter maps them back to LogSeverity, see [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/4caace7de8789f91dc3ddcdbcd328db002c44abb/exporter/collector/README.md#:~:text=The%20logging%20exporter%20also%20supports%20the%20full%20range%20of%20GCP%20log%20severity%20levels%2C%20which%20differ%20from%20the%20available%20OpenTelemetry%20log%20severity%20levels.%20To%20accommodate%20this%2C%20the%20following%20mapping%20is%20used%20to%20equate%20an%20incoming%20OpenTelemetry%20SeverityNumber%20to%20a%20matching%20GCP%20log%20severity%3A) and [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/types/severity.md).